### DISCUSSION TITLE
Claim Blocks Accrued Per Hour

### DISCUSSION THREAD

**Discussion by ghost** [GriefPrevention/GriefPrevention#1722] [General]
I thought that Claim Blocks Accrued Per Hour would add X claim blocks per hour. However, after testing it I confirmed that it is adding a random number of claim blocks per hour (11 and 13) in my 2 tests. How can I give a player 80 claim blocks per hour and not if they are idle for 10 minutes.  Also, if I want a claim to extend from bedrock to build limit, what would I set ExtendIntoGroundDistance to?

My config:
https://paste.gg/p/anonymous/4d65ffa36b9f45c69adfde7984dffdec

**Comment by ghost** [comment]
Bump!

I'm still looking for answer if someone could please help me with this.

**Comment by Jikoo** [comment]
Claim blocks accrued per hour are delivered in 5 minute increments which should total (barring rounding errors) your total claim blocks per hour. You can enable debug logging to monitor delivery. If you're not satisfied with GP's AFK detection system you can write an addon to control whether or not they are considered AFK during the AccrueClaimBlocksEvent using more accurate external tracking.

Claims always go to world height. If you want them to always go to bedrock as well, I recommend an ExtendIntoGroundDistance that is the same as your total world height.
